package.json

```diff
@@ -4,6 +4,9 @@
   "type": "module",
   "repository": "https://github.com/BrightspaceUI/labs.git",
   "exports": {
+    "./components/opt-in-flyout.js": "./src/components/opt-in-flyout/opt-in-flyout.js",
+    "./components/opt-out-flyout.js": "./src/components/opt-in-flyout/opt-out-flyout.js",
+    "./components/opt-out-reason.js": "./src/components/opt-in-flyout/opt-out-reason.js",
```

Comment:
Following the pattern for controllers -- I think this is nice actually as we get the benefit of the directories for code organization but when consumers are importing there's less repetition.